pkg/common/consumer/consumer_handler.go

```diff
@@ -141,4 +163,13 @@ func (consumer *SaramaConsumerHandler) ConsumeClaim(session sarama.ConsumerGroup
 	return nil
 }
 
+func sessionClosed(session sarama.ConsumerGroupSession) bool {
+	select {
+	case <-session.Context().Done():
```

Comment:
you can use just `Context().Err() != nil`

Reply:
TIL... thanks!